proof/crefine/AARCH64/StateRelation_C.thy

```diff
@@ -114,6 +114,12 @@ definition cur_vcpu_relation ::
       Some acurvcpu \<Rightarrow> cvcpu = Ptr (fst acurvcpu) \<and> cvcpu \<noteq> NULL \<and> cactive = from_bool (snd acurvcpu)
     | None \<Rightarrow> cvcpu = NULL \<and> cactive = 0"
 
+definition cur_fpu_relation :: "machine_word option \<Rightarrow> tcb_C ptr \<Rightarrow> bool" where
+  "cur_fpu_relation akscurfpu cfpu \<equiv>
```

Comment:
uhh... akscurfpu? it's more like afpu_opt at this point

Reply:
This is copying `cur_vcpu_relation` just above it. If we're being really pedantic then it should probably be something with owner in the name, since it's not talking about the tcb not the fpu.